### PR TITLE
test: Update pixel references to Fedora 39 tasks container, fix tests for Chromium 119

### PR DESCRIPTION
**Pull request by martinpitt**

### CODE REVIEW

**Comment by martinpitt** [comment]
[testAuthorizedKeys](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19632-20231120-133836-80303161-fedora-39-other/log.html) is now consistently broken across all OSes, the others are known high-profile flakes. F39 also has a minor pixel update.

**Comment by martinpitt** [comment]
`No matching package to install: 'pcp-libs-devel'` build error on [rawhide/i386](https://dashboard.packit.dev/results/copr-builds/1163610) is brand new from today, affects every PR, and unrelated to this one. https://bodhi.fedoraproject.org/updates/FEDORA-2023-cf7eccd442 was pushed to testing one hour today, and it refers to some arch dependency issue. I figure that's broken somehow, I'll have a look tomorrow.

**Review by jelly — APPROVED** [review]

**Comment by martinpitt** [comment]
https://src.fedoraproject.org/rpms/pcp/c/fc70568e935f8f4c88937cbe8fc17db9995ae7d5?branch=rawhide removes pcp from i686, so we need to drop that arch, too.

File: test/common/testlib.py [hunk 1]
```diff
@@ -241,6 +241,11 @@ def open(self, href: str, cookie: Optional[Dict[str, str]] = None, tls: bool = F
             # Force a reload in this case, to make tests and the waitPageLoad below predictable
             # But that option has the inverse effect with Chromium (argh)
             opts["transitionType"] = "reload"
+        elif self.cdp.browser.name == 'chromium':
+            # Chromium also optimizes this away, but doesn't have a knob to force loading
+            # so load the blank page first
+            self.cdp.invoke("Page.navigate", url="about:blank")
+            self.cdp.invoke("waitPageLoad", timeout=5)
```
Comment (mvollmer) [inline test/common/testlib.py:248]: So, I was thinking... can we predict whether the reload will be optimized away? Could we then skip the `waitPageLoad`?

**Review by mvollmer — APPROVED** [review]
I am fine with this, but it's quite hacky now... hopefully we can find something more elegant.

File: test/common/testlib.py (hunk 1 above)
Comment (martinpitt) [reply test/common/testlib.py:248]: Possibly, yes -- then we could skip  both steps. I still want to optimize this, but #1 priority is to get main back to green, so I landed this.

**Pull request merged**